### PR TITLE
Rename network_id to chain_id

### PR DESCRIPTION
**Pull request by karlb**
I intentionally did not change the `--network-id` CLI parameter, to keep
this as a simple refactoring instead of a breaking change. Maybe we can
even remove that parameter, as it can be derived from the ETH-RPC
endpoint.

Closes https://github.com/raiden-network/raiden/issues/1515

### CODE REVIEW

**Comment by ulope** [comment]
IIRC it was an intentional design decision to have the network-id parameter separate from the eth node url to prevent "accidents".

File: raiden/tests/unit/test_cli.py [hunk 1]
```diff
@@ -70,6 +70,7 @@ def test_run_error_reporting(cli_runner, monkeypatch):
     for exception, code in caught_exceptions.items():
         monkeypatch.setattr(cli, "run_services", mock_raises(exception))
         result = cli_runner(cli.run, "--accept-disclaimer")
+        print(result)
```
Comment (palango) [inline raiden/tests/unit/test_cli.py:73]: Looks like this one shouldn't be here.

File: raiden/ui/cli.py [hunk 2]
```diff
@@ -190,10 +191,10 @@ def options(func: Callable) -> Callable:
                 '"rinkeby" - network id: 4\n'
                 '"goerli" - network id: 5\n'
                 '"kovan" - network id: 42\n'
-                '"<NETWORK_ID>": use the given network id directly\n'
+                '"<CHAIN_ID>": use the given network id directly\n'
```
Comment (palango) [inline raiden/ui/cli.py:194]: I'd also rename the documentation text here.

**Review by palango — APPROVED** [review]
Nice, makes stuff more consistent! Two minor nits in the comments.

**Pull request merged**